libs/ledger/src/tx_status_http_interface.cpp

```diff
@@ -35,6 +35,25 @@ using fetch::variant::Variant;
 namespace fetch {
 namespace ledger {
 
+namespace {
+Variant toVariant(Digest const &digest, TransactionStatusCache::TxStatus const &tx_status)
+{
+  auto retval{Variant::Object()};
+  retval["tx"]     = ToBase64(digest);
+  retval["status"] = ToString(tx_status.status);
+  auto contr_ex_res_v{Variant::Object()};
```

Comment:
Not sure if we need this extra object level here. I think it makes more sense to have a single level as a return value